Makefile

```diff
@@ -340,6 +340,25 @@ $(TEST_OBJ_PATH)/%.opp: $(TEST_SRC_PATH)/%.cpp
 	@echo -n "compile "
 	@$(BUILD_CMD)
 
+FMT_FIXUP += $(THEMIS_TEST_FMT_FIXUP) $(SOTER_TEST_FMT_FIXUP)
```

Comment:
1. is it only for tests? shall we name it `FMT_TESTS_SETUP`?
2. is it ok to have these in main makefiles, maybe move to `tests/test.mk`?

Reply:
@vixentael,

> 1. is it only for tests? shall we name it `FMT_TESTS_SETUP`?

`FMT_FIXUP` is the variable which gathers all files to be formatted. We can introduce an intermediate variable to keep `THEMIS_TEST_FMT_FIXUP` and `SOTER_TEST_FMT_FIXUP` but it won't increase readability so I think it's redundant.

> 2. is it ok to have these in main makefiles, maybe move to `tests/test.mk`?

I have placed these lines in the main makefile because that's where compilation rules for test binaries are located. However, it really makes sense to put them into tests/test.mk. How about moving both new formatting rules _and_ existing build rules for tests there?

Reply:
@vixentael I've moved the formatting setup as well as compilation to `tests/test.mk`.

Reply:
thanks! my inner perfectionist feels much better now. Also naming `FMT_FIXUP` has more sense now when places into `tests.mk`